egs2/TEMPLATE/asr1/path.sh

```diff
@@ -1,7 +1,7 @@
 MAIN_ROOT=$PWD/../../..
 KALDI_ROOT=$MAIN_ROOT/tools/kaldi
 
-export PATH=$PWD/utils/:$KALDI_ROOT/tools/openfst/bin:$PATH
+export PATH=$PWD/utils/:$KALDI_ROOT/tools/openfst/bin:$KALDI_ROOT/tools/sctk/bin:$PATH
```

Comment:
```suggestion
export PATH=$PWD/utils/:$KALDI_ROOT/tools/openfst/bin:$PATH
```

Please not edit this.